website/templates/emails/welcome.html.mako

```diff
@@ -11,35 +11,46 @@
 
 Hello ${user.fullname},<br>
 <br>
-Thank you for verifying your OSF account, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
+Thank you for verifying your on OSF, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:
 <br>
+
+<h4>Select your storage location</h4>
+Files can be stored in a location you specify from the available geographic regions for new projects. <a href="https://osf.io/settings/account/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Set storage location.</a><br>
```

Comment:
Can we update the link to anchor to the part of the settings page where you set your default region?